homeassistant/components/camera/arlo.py

```diff
@@ -158,15 +161,48 @@ def disable_motion_detection(self):
         self._motion_status = False
         self.set_base_station_mode(ARLO_MODE_DISARMED)
 
+    @staticmethod
+    def clean_attr(attr):
```

Comment:
Instead to set UNKNOWN, filter that attribute out

Reply:
@pvizeli Are attributes dynamic? If an attribute becomes available after a refresh, can it be added, or is the list of attributes set in stone once the component is initialized?  The reason for setting to UNKNOWN is that the data may not be readily available when the component is initialized. The values would get populated after an update pulls from the REST endpoints.